Rakefile

```diff
@@ -1,15 +1,17 @@
 require 'rubygems'
 require 'bundler'
+require_relative 'ext/ext_helper'
 
-Bundler::GemHelper.install_tasks
 
 def ext_dependencies(name)
-  FileList["ext/#{name}/**/*"].reject { |file| file =~ /\b(Release|Debug)\b/ }
+  FileList["ext/#{name}/**/*"].reject { |file| file =~ /\b(Release|Debug|x86Release|x86Debug|x64Release|x64Debug)\b/ }
```

Comment:
Would it break anything if we would remove `\b` from the regexp so that we could simplify it to the following?
```ruby
FileList["ext/#{name}/**/*"].reject { |file| file =~ /(Release|Debug)/ }
```

Reply:
No this change would not break anything. It has been added, thanks